Core/src/org/sleuthkit/autopsy/contentviewers/MediaViewImagePanel.java

```diff
@@ -172,9 +172,8 @@ private void showErrorNode(String errorMessage, AbstractFile file) {
      * Show the contents of the given AbstractFile as a visual image.
      *
      * @param file image file to show
-     * @param dims dimension of the parent window (ignored)
      */
-    void showImageFx(final AbstractFile file, final Dimension dims) {
```

Comment:
removing this unnecessary parameter makes the import for java.awt.Dimension also unnecessary.